src/v2/components/AddBlock/index.tsx

```diff
@@ -1,28 +1,28 @@
-import React, { PureComponent } from 'react';
-import styled from 'styled-components';
-import { graphql } from 'react-apollo';
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import { graphql } from 'react-apollo'
 
-import Box from 'v2/components/UI/Box';
-import Text from 'v2/components/UI/Text';
-import Link from 'v2/components/UI/Link';
-import { FilledButton } from 'v2/components/UI/Buttons';
-import { Textarea } from 'v2/components/UI/Inputs';
-import DropZoneUploader from 'v2/components/UI/DropZoneUploader';
+import Box from 'v2/components/UI/Box'
+import Text from 'v2/components/UI/Text'
+import Link from 'v2/components/UI/Link'
+import { FilledButton } from 'v2/components/UI/Buttons'
+import { Textarea } from 'v2/components/UI/Inputs'
+import DropZoneUploader from 'v2/components/UI/DropZoneUploader'
 
-import createBlockMutation from 'v2/components/AddBlock/mutations/createBlock';
+import createBlockMutation from 'v2/components/AddBlock/mutations/createBlock'
```

Comment:
Prob goes without saying but the VSCode prettier plugin hooks into settings.json and formats on save

Reply:
Yeah I have that on